src/java.base/share/classes/java/lang/reflect/Executable.java

```diff
@@ -55,6 +55,11 @@
     @SuppressWarnings("deprecation")
     Executable() {}
 
+    @SuppressWarnings({"rawtypes"})
+    static final TypeVariable[] NO_TYPE_VARS = new TypeVariable[0];
+
+    static final Class<?>[] NO_TYPES = new Class<?>[0];
```

Comment:
Less concerning since it doesn't add extra classloading, but I note that we have similar constants in a few places, including `j.l.Class.EMPTY_CLASS_ARRAY` and j.l.r.ProxyGenerator.EMPTY_CLASS_ARRAY`. Perhaps would make sense to consolidate these into a single place/instance.

Reply:
A consolidated location like a non-instantiable java.lang.reflect.EmptyHolder class with static fields? (Exact name subject to bike shedding.)